CompassSDK/Tick/SendTik.swift

```diff
@@ -18,7 +18,7 @@ struct TikApiCall: ApiCall {
     let baseUrl: URL?
     let type: ContentType
     
-    init(baseUrl: URL? = Bundle.main.compassEndpoint, params: [String : Any], path: String, type: ContentType) {
+    init(baseUrl: URL? = TrackingConfig.shared.endpoint, params: [String : Any], path: String, type: ContentType) {
```

Comment:
use Global config new class in order to be able to configure tracker via info.plist or static method